lib/src/bloc/dice_bloc.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:async';
+import 'dart:math';
+import '../screen/dice/dice_event.dart';
+import 'package:first_flutter_app/src/bloc/bloc.dart';
+
+class DiceBloc extends Bloc {
+  final int maxNumDice = 6;
+  int _diceRoll = 0;
+  final _diceStateController = StreamController<int>();
+
+  StreamSink<int> get _inDiceRoll => _diceStateController.sink;
+
+  Stream<int> get diceRoll => _diceStateController.stream;
+  final _diceEventController = StreamController<DiceEvent>();
+
+  Sink<DiceEvent> get diceEventSink => _diceEventController.sink;
```

Comment:
I wouldnt provide a way from clients of this bloc to add items to the sink, we should eb exposing a mothod just to trigger the event we want, in that way we can control the input, makes sense ?

Reply:
Done.